test/services/sync/strategies/consumerActions/ClassAction.test.js

```diff
@@ -0,0 +1,78 @@
+const chai = require('chai');
```

Comment:
Like @tofixx expected, at the moment we have no handling for malformed data that come from queue. 
At the moment i think we can skip it. Because only one instance put the data in the queue. But we should add a comment to each action test that we must do it later. 
If we decouple the writen object in producer code and use a interface for it, maybe as a part of the support / shd producer -> task.